packages/v4/patternfly-docs/components-data.json

```diff
@@ -0,0 +1,354 @@
+{
+  "about-modal": {
+    "illustration": "./images/component-illustrations/./images/component-illustrations/about-modal.png",
+    "summary": "An <b>about modal</b> displays information about an application like product version number(s), as well as any appropriate legal text."
+  },
+  "accordion": {
+    "illustration": "./images/component-illustrations/accordion.png",
+    "summary": "An <b>accordion</b> is an interactive container that expands and collapses to hide or reveal nested content. It takes advantage of progressive disclosure to help reduce page scrolling, by allowing users to choose whether they want to show or hide more detailed information as needed."
+  },
+  "action-list": {
+    "illustration": "./images/component-illustrations/action-list.png",
+    "summary": "An <b>action list</b> is a group of actions with set spacing."
```

Comment:
Is it more accurate to describe an action list as a group of actions, controls, or buttons with set spacing?